nemo/core/classes/modelPT.py

```diff
@@ -1019,3 +1226,13 @@ def __is_model_being_restored() -> bool:
     def __set_model_restore_state(is_being_restored: bool):
         global _MODEL_IS_RESTORED
         _MODEL_IS_RESTORED = is_being_restored
+
+    @staticmethod
+    def __set_eff_save(use_eff_save: bool):
+        global _MODEL_EFF_SAVE
+        _MODEL_EFF_SAVE = use_eff_save
+
+    @staticmethod
+    def __use_eff_save() -> bool:
```

Comment:
Same here, remove the leading __

Reply:
wasn't sure how/where you want to use it, but ok, you are right, let's make it "public"

Reply:
done